openpype/plugins/publish/extract_color_transcode.py

```diff
@@ -0,0 +1,325 @@
+                view = output_def["view"] or colorspace_data.get("view")
+                display = (output_def["display"] or
+                           colorspace_data.get("display"))
+                # both could be already collected by DCC,
+                # but could be overwritten
+                if view:
+                    new_repre["colorspaceData"]["view"] = view
+                if display:
+                    new_repre["colorspaceData"]["display"] = display
+
+                files_to_convert = self._translate_to_sequence(
+                    files_to_convert)
+                for file_name in files_to_convert:
+                    input_path = os.path.join(original_staging_dir,
+                                              file_name)
+                    output_path = self._get_output_file_path(input_path,
+                                                             new_staging_dir,
+                                                             output_extension)
+                    convert_colorspace(
+                        input_path,
+                        output_path,
+                        config_path,
+                        source_colorspace,
+                        target_colorspace,
+                        view,
+                        display,
+                        self.log
+                    )
+
+                # cleanup temporary transcoded files
+                for file_name in new_repre["files"]:
+                    transcoded_file_path = os.path.join(new_staging_dir,
+                                                        file_name)
+                    instance.context.data["cleanupFullPaths"].append(
+                        transcoded_file_path)
+
+                custom_tags = output_def.get("custom_tags")
+                if custom_tags:
+                    if not new_repre.get("custom_tags"):
+                        new_repre["custom_tags"] = []
+                    new_repre["custom_tags"].extend(custom_tags)
+
+                # Add additional tags from output definition to representation
+                for tag in output_def["tags"]:
+                    if not new_repre.get("tags"):
+                        new_repre["tags"] = []
+                    if tag not in new_repre["tags"]:
+                        new_repre["tags"].append(tag)
+
+                    if tag == "review":
+                        added_review = True
+
+                instance.data["representations"].append(new_repre)
+                added_representations = True
+
+            if added_representations:
+                self._mark_original_repre_for_deletion(repre, profile,
+                                                       added_review)
+
+        for repre in tuple(instance.data["representations"]):
+            tags = repre.get("tags") or []
+            if "delete" in tags and "thumbnail" not in tags:
+                instance.data["representations"].remove(repre)
```

Comment:
@kalisp nice! It does work to have a "png" output, although you end up with "extra" PNGs. Ideally we need to introduce another tag that can delete the output later at the review extraction stage. Not sure what to call it, maybe `Delete output at review` or `Delete output later`. You can simulate this by having the tag `Delete output` and comment out these lines.

![Capture](https://user-images.githubusercontent.com/1860085/218533424-ee67003d-e4c4-43a9-b110-ef3f5a9c962e.PNG)

Reply:
Implemented logic to keep newly added representations for `extract_review`. 

We should design probably better way, but we need to define what "Delete output later" means.

Your Settings should allow create review, but delete source transcoded .pngs

Reply:
Noice! Tested and it works!